imports/components/@primitives/UI/live/index.js

```diff
@@ -75,18 +77,34 @@ class LiveWithoutData extends Component {
     }
   }
 
+  getLink() {
+    const { embedCode } = this.props.live;
+
+    // create beta link
+    const shouldShowBetaLink = (
+      this.props.person
+      && !this.props.person.authLoading
+      && this.props.person.authorized
+    );
+    return shouldShowBetaLink ? `/wowza/${embedCode}` : `/video/${embedCode}`;
```

Comment:
Right now, this is giving me the `doesn't match any routes` error because of the changes to the route in `pages/video/wowza.js`